package.json

```diff
@@ -14,7 +14,7 @@
     "cors": "^2.8.5",
     "express": "^4.16.4",
     "swagger-ui-express": "^4.1.4",
-    "yargs": "^8.0.2"
+    "yargs": "^15.4.1"
```

Comment:
yargs over v12 is not compatible with node v6, if we want to make this more general

Reply:
downgraded version